ppr-ui/src/components/mhrTransfers/BusinessSearchAutocomplete.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <v-card v-if="showAutoComplete" class="auto-complete-card" elevation="5">
+    <v-row no-gutters justify="center">
+      <v-col no-gutters cols="12">
+        <v-list v-if="autoCompleteResults && autoCompleteResults.length > 0" class="pt-0">
+          <v-list-item-group v-model="autoCompleteSelected">
+            <v-list-item
+              v-for="(result, i) in autoCompleteResults"
+              :key="i"
+              class="pt-0 pb-0 pl-3 auto-complete-item"
+              :class="{ disabled: isSPGP(result.legalType) }"
```

Comment:
Possibly a left over condition from the port. `isSPGP`? 
Might be able to remove class binding

Reply:
I could be wrong though, is this intentional?

Reply:
Yes, this is a requirement to disable SP GP business types.

Reply:
Thanks for the explanation! I wasn't sure at first glance if intentional.